insight-rest-objects/src/main/java/org/nuxeo/ai/sdk/objects/Statistic.java

```diff
@@ -25,6 +25,17 @@
 
 /**
  * A POJO representation of a Dataset Statistic
+ *
+ * <code>
+ * {
+ *   "id": "total",
+ *   "field": "total",
+ *   "type": "total",
+ *   "aggType": "total",
+ *   "numericValue": 10,
+ *   "value": null
+ * }
```

Comment:
Maybe add a second representation with something on the value, to see its format.